di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/handlers/CoreStubHandler.java

```diff
@@ -147,12 +147,18 @@ private void setQuestions() {
                         handlerHelper.exchangeCodeForToken(
                                 authorizationCode, credentialIssuer, state);
                 LOGGER.info("access token value: " + accessToken.getValue());
-                var userInfo =
+
+                var signedJWT =
                         SignedJWT.parse(
-                                        handlerHelper.getUserInfo(
-                                                accessToken, credentialIssuer, state))
-                                .getJWTClaimsSet()
-                                .toString();
+                                handlerHelper.getUserInfo(accessToken, credentialIssuer, state));
+
+                if (handlerHelper.isSignedUsingECDSA(signedJWT)) {
```

Comment:
As per the comments below invoke:
1) handlerHelper.checkSignatureFormat(signedJWT);`
2) `if (!handlerHelper.verifySignedJwt(signedJWT) { throw ... }`